app/models/shipit/user.rb

```diff
@@ -123,5 +129,20 @@ def identify_renamed_user!
     rescue Octokit::NotFound
       false
     end
+
+    def org_email?(email_address)
+      email_address.present? && email_address.split("@").last&.include?(org_domain)
```

Comment:
I don't think `include?` is a good idea. It means if you set org domain to `example.com`, then `example.com.rogue-haxor.net` will match.

`email_address.end_with?("@#{org_domain}")` would be much safer.

Reply:
Oh, that's much better.